src/client/components/Footer/style.css

```diff
@@ -0,0 +1,17 @@
+.footer {
+	width: 100%;
```

Comment:
Brakło ci jednej spacji

Reply:
Ty to jakoś poprawiłeś ? Bo ja nie widzę

Reply:
No właśnie tutaj też nie widzę, ale jak klikniesz nazwę pliku na górze to jak pojawi się pełny plik widać, że są 3 spacje zamiast 4. Ale nam w pracy też się zdarzało, że github dziwnie pokazywał z błędem odstępy - więc jeśli potwierdzisz, że w pliku jest ok, to mi wystarczy.

Reply:
jest git :+1: